packages/twitch-chat-client/src/ChatClient.ts

```diff
@@ -642,7 +647,7 @@ export default class ChatClient extends IRCClient {
 		/* eslint-disable no-restricted-syntax */
 		super({
 			connection: {
-				hostName: options.webSocket === false ? 'irc.chat.twitch.tv' : 'irc-ws.chat.twitch.tv',
+				hostName: options.hostName ?? (options.webSocket ? 'irc-ws.chat.twitch.tv' : 'irc.chat.twitch.tv'),
```

Comment:
The `=== false` was intended because it's supposed to behave differently on `undefined` (aka it should use websockets by default). While one could argue about that being okay (I don't think so), this is a breaking change and has no place in the minor version release that's currently being built.

`options.webSocket === false`:
not given => ws
true => ws
false => no ws

```options.webSocket```:
not given => **no ws**
true => ws
false => no ws

Reply:
Fixed.